Makefile

```diff
@@ -132,6 +132,10 @@ k8s-migrate:
 	$(shell kubectl --namespace ${DEIS_APP} get pods | grep ${DEIS_APP}-cmd | awk '{print $$1}') \
 	python manage.py migrate
 
+render-k8s-templates:
+	k8s/api-svc.yaml.template.sh > k8s/api-svc.yaml
+	rm k8s/api-svc.yaml.template.sh
```

Comment:
It didn't seem to make any difference whether or not it was removed. Maybe it's safer to keep the removal just so there's no possibility of confusing ``kubectl``? I'll leave it in for now.